plugins/modules/packaging/os/rpm_ostree_pkg.py

```diff
@@ -32,9 +32,17 @@
       choices: [ 'absent', 'present' ]
       default: 'present'
       type: str
+    apply_live:
+      description:
+      - Stage packages and overlay them on the running system tree.
+      - Corresponds to the C(--apply-live) option. Default behavior (C(no)); C(yes) overlays the requested package(s) to the live system.
+      required: false
+      default: "no"
```

Comment:
(`false` is default for `required`, so it's better to leave that line away.)
```suggestion
      default: false
```